docs/installation/install-livepeer/binary-release.md

```diff
@@ -40,19 +39,37 @@ mv livepeer-linux-amd64/* /usr/local/bin/
 
 ### Windows
 
-```bash
-# <RELEASE_VERSION> is the release version i.e. 0.5.23
+.tar
+
+```shell
+# <RELEASE_VERSION> is the release version, e.g. v0.5.29
 
-# First, fetch the latest release
+# Fetch the latest release .tar.gz 
 wget https://github.com/livepeer/go-livepeer/releases/download/<RELEASE_VERSION>/livepeer-windows-amd64.tar.gz
 
 # Next, extract it
-tar -zxvf livepeer-windows-amd64.tar.gz
+tar -zxvf li2
 
-# Finally, move it to the appropriate directory
-mv livepeer-windows-amd64/* /usr/local/bin/
+# Finally, move it to the appropriate directory, e.g. C:/Users/usr/Desktop/livepeer-folder
+move livepeer-windows-amd64 /*/usr/local/<folder-name>
```

Comment:
If it's for Windows, shouldn't we just have it like this?

```
move livepeer-windows-amd64 C:/Users/usr/Desktop/livepeer-folder/
```

Reply:
well, they can save it where ever they want, we aren't prescribing, that's why we made an e.g.

Reply:
Yes, but this description is dedicated to Windows. The `move` command is only for Windows. And in Windows, you don't have `/*/usr/local/<folder-name>` directory. So I'd change it to something that would work in Windows.

Reply:
@leszko what's an appropriate filepath for Windows? I'm more of a Mac guy so I'm not sure what to suggest here - do you have windows experience?

Reply:
For example `C:\Users\UserName\livepeer-folder`.

Reply:
That works, i will put that in the text instead of the line.

Reply:
@leszko i removed the .tar instructions -- we only use .zip for windows (unless there is a reason it should be there...)
Also, btw is the tar command for extracting on linux and darwin correct?
My research says  `tar -xzvf myfile.tar.gz` and our docs say `tar -zxvf livepeer-darwin-amd64.tar.gz`

Reply:
> 

It's the same. The parameter `-xzvf` is the same as `-zxvf`

Reply:
okay, we'll just leave it as is